go.mod

```diff
@@ -1,7 +1,9 @@
 module github.com/bwmarrin/discordgo
 
 require (
+	filippo.io/edwards25519 v1.0.0-beta.2 // indirect
 	github.com/gorilla/websocket v1.4.0
+	github.com/kevinburke/nacl v0.0.0-20201212185859-3ef16edae976
```

Comment:
this library implements nacl signing, and is a thin wrapper around `crypto/ed25519`.